### DISCUSSION TITLE
part-ii/2018-159/

### DISCUSSION THREAD

**Discussion by tripos-comments** [tripos-education/maths-tripos-questions#223] [Question comments]
Part II 2018 Waves: Paper 1, Section II, 39C 

https://questions.tripos.org/part-ii/2018-159/

**Comment by thenewtonium** [comment]
<details> <summary>[Spoilers]</summary>

First part is just stuff from notes.

For the rest of the qn, consider a spherical harmonic wave $r\phi = A e^{i\omega t} \sin(r\omega /c_0)$, (where the $\sin$ comes from regularity at $r=0$).

(Linearised) Kinematic boundary condition is $\dot{a} = u_r = \frac{\partial\phi}{\partial r}$ imposed at $r = a_0$.

From this we can find $a$, up to a constant (which we won't need to determine).

(Linearised) Dynamic boundary condition is $m\ddot{a} = -\kappa (a-a_0) + \tilde{p}(r=a_0)$. Or maybe with an extra $p_0$ term (though it seems as if the given restoring force already accounts for the equilibrium pressure), but it doesn't make a difference to the calculation whether you include it or not, since all it changes is the the integration constant in $a$.

Sub in our expressions for $\ddot{a}$ and $a$, and $\tilde{p} = -\rho_0\frac{\partial \phi}{\partial t}$. Since this condition must hold $\forall t$, compare coefficients of the $e^{i\omega t}$ terms and massage this into the equation they want.

For the last part I get that there are infinitely many solutions for $\theta$; $\frac{1}{\theta \cot \theta -1}$ basically just looks like $\frac{\tan \theta}{\theta}$ with the zeroes and asymptotes shifted a bit.

</details>
